libvast/vast/plugin.hpp

```diff
@@ -420,3 +385,23 @@ class plugin_ptr final {
     }
 
 #endif // !defined(VAST_ENABLE_STATIC_PLUGINS_INTERNAL)
+
+#define VAST_REGISTER_PLUGIN_1(name)                                           \
+  VAST_REGISTER_PLUGIN_5(name, ::vast::version::major, ::vast::version::minor, \
+                         ::vast::version::patch, ::vast::version::tweak)
+
+#define VAST_REGISTER_PLUGIN_2(name, major)                                    \
+  VAST_REGISTER_PLUGIN_5(name, major, 0, 0, 0)
+
+#define VAST_REGISTER_PLUGIN_3(name, major, minor)                             \
+  VAST_REGISTER_PLUGIN_5(name, major, minor, 0, 0)
+
+#define VAST_REGISTER_PLUGIN_4(name, major, minor, patch)                      \
+  VAST_REGISTER_PLUGIN_5(name, major, minor, patch, 0)
+
+#define VAST_REGISTER_PLUGIN(...)                                              \
+  VAST_PP_OVERLOAD(VAST_REGISTER_PLUGIN_, __VA_ARGS__)(__VA_ARGS__)
+
+#define VAST_REGISTER_PLUGIN_TYPE_ID_BLOCK(...)                                \
+  VAST_PP_OVERLOAD(VAST_REGISTER_PLUGIN_TYPE_ID_BLOCK_, __VA_ARGS__)           \
```

Comment:
This file is missing an include for `vast/detail/pp.hpp`.

Reply:
Good catch, totally missed that.